stories/Animations/ExampleStandard.js

```diff
@@ -45,6 +45,12 @@ class AnimatedExample extends React.Component {
       {id: 'small', value: 'Small - 150ms'},
       {id: 'medium', value: 'Medium - 200ms'},
       {id: 'large', value: 'Large - 300ms'},
+      {id: 'fast01', value: 'Large - 100ms'},
```

Comment:
I cannot find any usage of this example. If it is not used anymore maybe we don't need changes here.

Reply:
Agree, I added it by mistake so I will remove it.